chapter2/tests/mkcert/nginx.conf

```diff
@@ -0,0 +1,87 @@
+user nginx nginx;
+worker_processes 1;
+pid /run/nginx.pid;
+include /etc/nginx/modules-enabled/*.conf;
+
+events {
+	worker_connections 768;
+	# multi_accept on;
```

Comment:
If you don't need specific settings - don't keep them or commented, this would lead to a chaos later on during the maintenance phase.

Why he/she did comment it? Was it useful for something? For what?
People ask too many questions in such cases.